### PR TITLE
Refactored docstrings in mod:`manim.mobject.graphing.coordinate_systems`

### PR DESCRIPTION
**Pull request by Kiran-Raj-Dev**
## Further Information and Comments

This one is pretty big, but I didn't know how to break it up.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/mobject/graphing/coordinate_systems.py [hunk 1]
```diff
@@ -402,10 +383,22 @@ def construct(self):
         :class:`~.VGroup`
             A :class:`~.Vgroup` of the labels for the x_axis and y_axis.
 
-        See Also
+        .. seealso::
+            :class:`get_x_axis_label`
+            :class:`get_y_axis_label`
+
```
Comment (icedcoffeeee) [inline manim/mobject/graphing/coordinate_systems.py:389]: [This](https://manimce--2570.org.readthedocs.build/en/2570/reference/manim.mobject.graphing.coordinate_systems.CoordinateSystem.html#manim.mobject.graphing.coordinate_systems.CoordinateSystem.get_axis_labels) doesn't render correctly

File: manim/mobject/graphing/coordinate_systems.py [hunk 2]
```diff
@@ -575,29 +577,32 @@ def construct(self):
...
 
         Parameters
         ----------
         point
-            The point to which the horizontal line will be drawn.
-
+            A point on the scene.
         kwargs
-            Additional parameters to be passed to :class:`get_line_from_axis_to_point`
+            Additional arguments to be provided :meth:`CoordinateSystem.get_line_from_axis_to_point`.
```
Comment (icedcoffeeee) [inline manim/mobject/graphing/coordinate_systems.py:592]: I think I prefer the previous wording better, 🤔

File: manim/mobject/graphing/coordinate_systems.py [hunk 3]
```diff
@@ -1335,9 +1296,22 @@ def plot_derivative_graph(
     ) -> ParametricFunction:
         """Returns the curve of the derivative of the passed graph.
 
+        Parameters
+        ----------
+        graph
+            The graph for which the derivative will be found.
+        color
+            The color of the derivative curve.
+        **kwargs
```
Comment (icedcoffeeee) [inline manim/mobject/graphing/coordinate_systems.py:1305]:
```suggestion
        kwargs
```

This seems inconsistent throughout the file. I think it should be changed.

File: manim/mobject/graphing/coordinate_systems.py [hunk 4]
```diff
@@ -1744,6 +1668,25 @@ def construct(self):
         Whether or not to include the tips on both axes.
     kwargs : Any
```
Comment (icedcoffeeee) [inline manim/mobject/graphing/coordinate_systems.py:1669]:
```suggestion
    kwargs
```
Port over docs typehints to in-code ones?

**Review by icedcoffeeee — CHANGES_REQUESTED** [review]
Left a few comments; otherwise, great work!

File: manim/mobject/graphing/coordinate_systems.py (hunk 2 above)
Comment (Kiran-Raj-Dev) [reply manim/mobject/graphing/coordinate_systems.py:592]: I don't like it either and neither do I remember ever doing such a change 🤔

File: manim/mobject/graphing/coordinate_systems.py (hunk 4 above)
Comment (Kiran-Raj-Dev) [reply manim/mobject/graphing/coordinate_systems.py:1669]: Hmm, only `kwargs` were missed.
We haven't type hinted them anywhere, do you still want me to?

Comment (icedcoffeeee) [reply manim/mobject/graphing/coordinate_systems.py:1669]: Ah I see, then my comment is invalid. Plus, I don't think kwargs should be typehinted.

File: manim/mobject/graphing/coordinate_systems.py (hunk 1 above)
Comment (Kiran-Raj-Dev) [reply manim/mobject/graphing/coordinate_systems.py:390]: Admonitions must have two newline characters after any section. Changes done in [e4d6dd3](https://github.com/ManimCommunity/manim/pull/2570/commits/e4d6dd35b15007c1d1643cf2908b28d235f2f16d)

**Comment by Kiran-Raj-Dev** [comment]
Thanks for reviewing this icedtea(coffee?) !

**Review by Darylgolden — APPROVED** [review]
Thank you!

**Review by icedcoffeeee — APPROVED** [review]
LGTM, thanks!

**Review by naveen521kk — APPROVED** [review]
lgtm

**Pull request merged**